### PR TITLE
#43 removed organizer

### PR DESCRIPTION
**Pull request by adrianczekaj**

### CODE REVIEW

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs [hunk 1]
```diff
@@ -42,20 +40,19 @@ public ActionResult<List<DrawingResultDto>> GetAllForEvent([FromRoute] Guid even
...
 		}
 		
-		var @event = _eventRepository.Get(eventId);
-		var organizer = _organizerRepository.Get(@event.OrganizerId);
-		
-		var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x=>x.Id);
-		var body = "http://localhost:5036/DrawingResults/" + string.Join("\nhttp://localhost:5036/DrawingResults/", drawingResultIds);
-
-		_mail.Send($"{organizer.Email}", $"Links to drawing results '{@event.Name}'", $"{body}");
+		//TODO: fix - organizer is now person type
```
Comment (maciejczekaj) [inline GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]: fix this code

File: GiftGivingGenerator.API/Controllers/PersonsController.cs [hunk 2]
```diff
@@ -9,30 +9,19 @@ namespace GiftGivingGenerator.API.Controllers;
...
-		
-		_organizerRepository.Update(organizer);
-		return CreatedAtAction(nameof(CreatePerson), new {id = person.Id}, null);
 	}
 
 	[HttpGet("/Organizers/{organizerId}/Persons")]
 	public ActionResult<List<PersonDto>> GetPersonsByOrganizer([FromRoute] Guid organizerId)
 	{
-		var personsDto = _repository.GetPersonsByOrganizer(organizerId);
+		//TODO: write new method to GetPersonsByOrganizer
```
Comment (maciejczekaj) [inline GiftGivingGenerator.API/Controllers/PersonsController.cs]: fix this endpoint

**Review by maciejczekaj — CHANGES_REQUESTED** [review]

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs [hunk 3]
```diff
@@ -44,20 +46,19 @@ public ActionResult SendEmail([FromRoute] Guid eventId)
...
-		// var body = "http://localhost:5036/DrawingResults/" + string.Join("\nhttp://localhost:5036/DrawingResults/", drawingResultIds);
-		//
-		// _mail.Send($"{organizer.Email}", $"Links to drawing results '{@event.Name}'", $"{body}");
-		
+
+		var @event = _eventRepository.Get(eventId);
+		var organizer = _personRepository.Get(@event.OrganizerId);
+
+		var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x => x.Id);
+		var body = "http://localhost:5036/DrawingResults/" + string.Join("\nhttp://localhost:5036/DrawingResults/", drawingResultIds);
```
Comment (maciejczekaj) [inline GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]: - move the base host to app settings
- don't concatenate strings (use string interpolation instead)

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs [hunk 4]
```diff
@@ -44,20 +46,19 @@ public ActionResult SendEmail([FromRoute] Guid eventId)
...
-		// var @event = _eventRepository.Get(eventId);
-		// var organizer = _organizerRepository.Get(@event.OrganizerId);
-		//
-		// var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x=>x.Id);
-		// var body = "http://localhost:5036/DrawingResults/" + string.Join("\nhttp://localhost:5036/DrawingResults/", drawingResultIds);
-		//
-		// _mail.Send($"{organizer.Email}", $"Links to drawing results '{@event.Name}'", $"{body}");
-		
+
+		var @event = _eventRepository.Get(eventId);
```
Comment (maciejczekaj) [inline GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]: use `Get<T>(eventId)`

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs [hunk 5]
```diff
@@ -44,20 +46,19 @@ public ActionResult SendEmail([FromRoute] Guid eventId)
...
+		var organizer = _personRepository.Get(@event.OrganizerId);
+
+		var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x => x.Id);
```
Comment (maciejczekaj) [inline GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]:
```
var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x => $"http://localhost:5036/DrawingResults/{x.Id}");
var body = string.Join("\n", drawingResultIds);
```

File: GiftGivingGenerator.API/Servicess/MailService.cs [hunk 6]
```diff
@@ -15,8 +15,7 @@ public void Send(string to, string subject, string body)
 	{
 		MailMessage message = new MailMessage(_options.userName, to, subject, body);
 
-		System.Net.NetworkCredential basicCredential1 = new
-			System.Net.NetworkCredential(_options.userName, "_options.password");
+		System.Net.NetworkCredential basicCredential1 = new System.Net.NetworkCredential(_options.userName, _options.password);
```
Comment (maciejczekaj) [inline GiftGivingGenerator.API/Servicess/MailService.cs]: better use:
`var basicCredential1 = new System.Net.NetworkCredential(_options.userName, _options.password);`
or
`System.Net.NetworkCredential basicCredential1 = new (_options.userName, _options.password);`

**Review by maciejczekaj — CHANGES_REQUESTED** [review]

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs (hunk 3 above)
Comment (adrianczekaj) [reply GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]: done

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs (hunk 4 above)
Comment (adrianczekaj) [reply GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]: done

File: GiftGivingGenerator.API/Controllers/DrawingResultsController.cs (hunk 5 above)
Comment (adrianczekaj) [reply GiftGivingGenerator.API/Controllers/DrawingResultsController.cs]: removed sending email to organizer with all links to drawing results -> sending one link (to event)

File: GiftGivingGenerator.API/Servicess/MailService.cs (hunk 6 above)
Comment (adrianczekaj) [reply GiftGivingGenerator.API/Servicess/MailService.cs]: done

**Review by maciejczekaj — APPROVED** [review]

**Pull request merged**